dvc/command/remote.py

```diff
@@ -108,6 +108,34 @@ def run(self):
         return 0
 
 
+class CmdRemoteRename(CmdRemote):
+    def run(self):
+        conf = self.config.load_one(self.args.level)
+        self._check_exists(conf)
+
+        all_config = self.config.load_config_to_level("all")
```

Comment:
Should this be `load_config_to_level()` since `"all"` isn't a defined level? (unless using `"all"` is supposed to actually do something different than calling `load_config_to_level(level=None)`)

Reply:
`"all"` is more readable than `None`, this is the reason I used it. Both of them give the same result. As you agreed with `None`, then it is now a better choice for consistency with team code style.